app/components/crate-sidebar/install-instructions.js

```diff
@@ -0,0 +1,21 @@
+import Component from '@glimmer/component';
+
+export default class InstallInstructions extends Component {
+  get cargoInstallCommand() {
+    return this.args.exactVersion
+      ? `cargo install ${this.args.crate}@${this.args.version}`
+      : `cargo install ${this.args.crate}`;
+  }
+
+  get cargoAddCommand() {
+    return this.args.exactVersion
+      ? `cargo add ${this.args.crate}@=${this.args.version}`
+      : `cargo add ${this.args.crate}`;
+  }
```

Comment:
May I ask why we need to use `cargo add crate-name@=x.x.x` to install the library and `cargo install crate-name@x.x.x` to install the binary? Why do we need to use `=` for adding a library?

Reply:
my understanding is that for `cargo add` the version number includes an implicit `^` constraint, while with `cargo install` my local testing showed that an implicit `=` constraint was used.

Reply:
For library, in some cases, it's necessary to add `@=` to install a specific version, as initially reported in issue #7884.
On the other hand, it seems that binary is always installed with the specified version.

Reply:
Oh, I see. I hadn't noticed that. Thank you!